scripts/checkLinks.ts

```diff
@@ -124,11 +118,8 @@ function checkLinks(): void {
         return; // Skip external links
       }
 
-      const siteLinks = ['/install', '/images', '/profile'];
-      for (const siteLink of siteLinks) {
-        if (link.startsWith(siteLink)) {
-          return; // Skip site links
-        }
+      if (!link.startsWith('/docs')) {
+        return; // Skip site links
```

Comment:
I feel like this probably breaks the "checking image URLs for validity" logic.. but we were explicitly skipping `/images` before anyway, so..?

Reply:
Yes, we're not checking image links for validity because they are shipped with the website. There is a long-standing issue to implement images in this repo, but we haven't done it yet.

Reply:
okay great :+1: then this isn't a regression